### ISSUE TITLE
blkid -o export does not return on first invocation but returns on second

### ISSUE THREAD

**Issue by vikramkhatri** [karelzak/util-linux#1377]
I am seeing a strange issue in `blkid -o export /dev/disk/by-label/root` command that it does not work on the first invocation but it works on the second execution especially when the boot process drops to the emergency shell. This is causing a system OS script to fail and I have raised an issue with RedHat.

For example:

```
For example:

.:/# rootpath=/dev/disk/by-label/root
:/# blkid -o export ${rootpath} --.> Does not show any output when run
:/# blkid -o export ${rootpath} --> Shows output on second run
DEVNAME=/dev/disk/by-label/root
LABEL=root
UUID=910678ff-f77e-4a7d-8d53-86f2ac47a823
TYPE=xfs
:/# blkid -o export ${rootpath} --.> Does not show any output
:/# blkid -o export ${rootpath} --> Shows output on second run
DEVNAME=/dev/disk/by-label/root
LABEL=root
UUID=910678ff-f77e-4a7d-8d53-86f2ac47a823
TYPE=xfs
:/#
```
`blkid -o export ${rootpath}` --> Works sometimes and it does not work at times as well.

Please note: I do not see this issue in a system that has booted fully but only on a system that has dropped to the emergency shell.

**Comment by karelzak** [comment]
Strange. Are you sure the symlink /dev/disk/by-label/root is there? ;-)

Anyway, please try 

   `# LIBBLKID_DEBUG=all blkid -o export /dev/disk/by-label/root`

to get more details. Thanks.

**Comment by svang-app** [comment]
We were able to find the root cause with the help of RedHat support. The underlying disk was RAID 1 at the hardware level. Somehow one disk was not having the label while the other disk had the label. After we broke the mirror - it worked well. RedHat used the above method `LIBBLKID_DEBUG=all` to find the root cause.

`blkid -o export ${rootpath}` worked once and then it did not work on the second invocation but again worked in the next one. At least the root cause of the problem is resolved and I have to yet resolve the hardware issue. 

Thank YOU.
